projects/core/src/lib/constants/default-element-predicate.ts

```diff
@@ -1,4 +1,5 @@
 import {MaskitoElementPredicate} from '../types';
 
 export const MASKITO_DEFAULT_ELEMENT_PREDICATE: MaskitoElementPredicate = e =>
-    e.querySelector('input,textarea') || (e as HTMLInputElement | HTMLTextAreaElement);
+    e.querySelector<HTMLInputElement | HTMLTextAreaElement>('input,textarea') ||
```

Comment:
Could we revert these lines ? 

See https://github.com/Tinkoff/maskito/pull/316#discussion_r1203573546

Reply:
Yes 👍 with the recent type changes, we can revert these. 

Addressed here: https://github.com/Tinkoff/maskito/pull/316/commits/e2b89eb84b7a5b1d483b287e45b7197466aae577